releasenotes/notes/ddtracemethods-56cbda5e38a8e15c.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    tracer: Adds support for DD_TRACE_METHODS. This feature enables the
+      specification of custom methods to be instrumented by the tracer when using ``ddtrace-run``.
+      See the configuration documentation for more information.
```

Comment:
would it be nice to link to the configuration documentation? We can just guess correctly what the header would be right? https://ddtrace.readthedocs.io/en/stable/configuration.html#DD_TRACE_METHODS

Reply:
Ah yes, it would! I will add a link to the docs in the release note.